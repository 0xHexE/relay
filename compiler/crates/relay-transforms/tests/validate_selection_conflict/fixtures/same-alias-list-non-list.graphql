# expected-to-throw
fragment TestFragment on Node {
    ... on Feedback {
        actor {
            name
        }
    }
    ... on Comment {
        actor: actors {
            name
        }
    }
}
