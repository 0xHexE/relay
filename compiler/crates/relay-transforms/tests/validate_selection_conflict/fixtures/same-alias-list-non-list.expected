==================================== INPUT ====================================
# expected-to-throw
fragment TestFragment on Node {
    ... on Feedback {
        actor {
            name
        }
    }
    ... on Comment {
        actor: actors {
            name
        }
    }
}
==================================== ERROR ====================================
✖︎ Field 'actor' is ambiguous because it references fields with different types: 'actor' with type 'Actor' and 'actors' with type '[Actor]'

  same-alias-list-non-list.graphql:4:9
    3 │     ... on Feedback {
    4 │         actor {
      │         ^^^^^
    5 │             name

  ℹ︎ the other field

  same-alias-list-non-list.graphql:9:16
    8 │     ... on Comment {
    9 │         actor: actors {
      │                ^^^^^^
   10 │             name
